speedscale/README.md

```diff
@@ -0,0 +1,54 @@
+# Agent Check: Speedscale
+
+## Overview
+
+This integration publishes the results from [Speedscale][1] into Datadog.
+
+## Setup
+
+### Configuration
+
+In order to utilize the integration you need to capture the following from Datadog:
+
+* [API Key][2] - An API key is required by the Datadog Agent to submit metrics and events to Datadog
+
+A best practice is to save this into environment variables like so:
+
+```
+export DDOG_API_KEY=0
```

Comment:
Once the user has saved their API key, where would they enter it to connect to Speedscale? It would be helpful to have some step by step instructions here.

Reply:
Thank you, I reworked this section to keep all the configuration in one place. Right now this is done through continuous integration and that is specifically called out in my latest update.